ui/lib/apps/SlowQuery/utils/useSlowQueryTableController.ts

```diff
@@ -120,26 +115,18 @@ export default function useSlowQueryTableController(
     [visibleColumnKeys]
   )
 
-  const cacheKey = useMemo(() => {
-    const {
-      schemas,
-      digest,
-      limit,
-      plans,
-      searchText,
-      timeRange,
-    } = queryOptions
-    const { desc, orderBy } = orderOptions
-    const cacheKey = `${schemas.join(',')}_${digest}_${limit}_${plans.join(
-      ','
-    )}_${searchText}_${stringifyTimeRange(
-      timeRange
-    )}_${desc}_${orderBy}_${selectedFields}`
-    return cacheKey
-  }, [queryOptions, orderOptions, selectedFields])
+  const isOnMount = useRef(true)
+  useEffect(() => {
+    if (!isOnMount.current) {
+      cacheMgr?.remove()
+    }
+    if (isOnMount.current) {
+      isOnMount.current = false
+    }
+  }, [queryOptions, orderOptions, selectedFields, cacheMgr])
```

Comment:
the queryOptions changes several times when in mount stage, which is caused by `setQueryOptions` method.